internal/nodes/dequeue_ingestion.go

```diff
@@ -571,6 +580,9 @@ func handleL4SharedVipService(namespacedVipKey, key string, fullsync bool) {
 	for i, serviceNSName := range serviceNSNames {
 		svcNSName := strings.Split(serviceNSName, "/")
 		svcObj, err := utils.GetInformers().ServiceInformer.Lister().Services(svcNSName[0]).Get(svcNSName[1])
+		if !lib.ValidateSvcforClass(key, svcObj) {
```

Comment:
here we should not process any of L4 if one of the shared service fails with vaildateSvcClas

Reply:
I've replaced `continue` with `break`